lib/service/services/authentication/service/helpers/handle_auth_response.dart

```
@@ -7,7 +7,21 @@ AuthenticationModel handleAuthResponse(Map<String, dynamic> json) {
 
   if (result == 0) {
     return response.authModel!;
-  } else if (result == -9) {
+  } else {
+   _validateResultResponse(result);
+   throw '';
```

` throw 'Something went wrong'`; instead of empty throw

---

will be discussed in DM